bindings/java/iota-client-java/examples/ExampleProject/src/node_api_core/CreateBlock.java

```diff
@@ -0,0 +1,23 @@
+package node_api_core;
+
+import com.google.gson.JsonObject;
+import org.iota.Client;
+import org.iota.types.Block;
+import org.iota.types.ClientConfig;
+import org.iota.types.ClientException;
+import org.iota.types.ids.BlockId;
+
+public class CreateBlock {
+
```

Comment:
Your use of blank lines in the examples is very inconsistent, would be nice if you could harmonise a bit

Reply:
fixed